### DISCUSSION TITLE
Suricata and Zeek

### DISCUSSION THREAD

**Discussion by lmb2100** [Security-Onion-Solutions/securityonion#10860] [General]
I have an air-gapped distributed deployment with search manager and 2 sensors for training with simulated attacks. Can I run zeek and suricata at the same time? I only see Zeek events ( I do have zeek set as the metadata) and was wondering if there is more configuration I would need to do in order to see suricata logs? Or am I not understanding the use case of these modules? Thanks!

**Comment by cm-ops** [comment]
You can take a look here https://docs.securityonion.net/en/latest/network.html to see information on `Suricata` and `Zeek`. If you are running `Zeek` as metadata you probably have `Suricata` set as your signature-based IDS. You can also just run `Suricata` for both.

For the logs, do you see any logs on your sensors at `/nsm/suricata/`?

**Comment by lmb2100** [comment]
I didn't see anything under  nsm/suricata. Does that mean it is not
running? On kibana I only see zeek sources

**Reply by vanh20** [reply]
If you have configured Suricata as your IDS, than you should see logs at /nsm/suricata/. Have you disabled Suricata as described [here](https://docs.securityonion.net/en/latest/suricata.html#disabling)?
